### DISCUSSION TITLE
How do I find the path for Mastadon installed in a Docker container?

### DISCUSSION THREAD

**Discussion by TruthTriumphs** [mastodon/mastodon#24784] [Q&A]
I have a Linode account for virtual linux server
Installed Cloudron to use for installing Mastadon
Cloudron installed Mastadon in a Docker container
Mastadon is installed & running at my domain
I have SFTPed in to my Linode volume and cannot find the beginning file path for the Mastadon installation ... specifically looking for js and image files.
I've read Mastadon docs but I find nothing, no matter how much I drill into the file structure at my Linode volume, that matches the file path where I expect to find the core files for my Mastadon installation.
Can you give me the details for where I should start looking in my Linode volume for Mastadon core installed files?
I can find folders referring to Cloudron and Docker but diving into them and every other folder I see does not reveal the actual Mastadon files by name or the file paths I see mentioned in the Mastadon docs ... such as /app/data/

**Comment by circuitsunfish** [comment]
`docker exec -it live_web_1 bash` should get you into the container where Mastodon's files are being executed from. From there you should be able to inspect the container's file structure.

For a list of docker containers, if your container structure does not match the one given by Linode's app marketplace, use `docker ps`.
